### DISCUSSION TITLE
GPU Support

### DISCUSSION THREAD

**Discussion by k-e-r-n-e-l-p-a-n-i-c** [jupyterlite/jupyterlite#802] [Q&A]
Hello,

Is it possible to make use of GPU's in JupyterLite? If yes, can you please point me to the correct documentation?

Regards,

**Comment by bollwyvl** [comment]
I know of no examples, yet, for any of the stock kernels in this repo.

Here's a quick hack:

https://gist.github.com/bollwyvl/29f0e8011fe58c7af0915583c8e7d45e

I don't know how _useful_ this is, per se, but at least it's _possible_.

**Reply by k-e-r-n-e-l-p-a-n-i-c** [reply]
Thanks @bollwyvl .  I wasn't aware of gpu.js until now. This really looks interesting to me.

**Reply by bollwyvl** [reply]
sure. there are _many_ ways to get access to the GPU primitives, but as with any web "standard," the trick is doing it in a productive way that is robust vs all of the implementation details. Luckily we're basically down to chromium, firefox and safari (ugh), but each of those engines weighs in on feature vs security tradeoffs that apps/libraries like jupyterlite can't always account for seamlessly. Good luck, please report back any findings!
